src/hdmf/data_utils.py

```diff
@@ -599,6 +599,12 @@ def __init__(self, **kwargs):
         data = popargs('data', kwargs)
         self.__data = data
 
+    def get_io_params(self):
+        """
+        Returns a dict with the I/O parameters specifiedin in this DataIO.
+        """
+        return dict()
```

Comment:
Is this right? Just return an empty `dict`?

Reply:
The base DataIO class offers no ability to specify parameters, so there's not really anything to return